client/src/templates/Challenges/utils/frame.ts

```diff
@@ -106,60 +111,72 @@ export const runTestInTestFrame = async function (
 };
 
 const createFrame =
-  (document: Document, id: string, title?: string) => (ctx: Context) => {
+  (document: Document, id: string, title?: string) =>
+  (frameContext: Context) => {
```

Comment:
I changed `ctx` to `frameContext` because I was having difficulty noticing them sometimes. I am happy to revert it back

Reply:
👍